blossom/templates/website/partials/base.partial

```diff
@@ -23,14 +23,14 @@
     <meta name="Description"
           content="The Grafeas Group is a 501(c)(3) working on increasing accessibility on the internet though crowdsourcing and gamification.">
 
-    <script src="https://code.jquery.com/jquery-3.4.1.min.js"
-            crossorigin="anonymous"></script>
-    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
-          integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.1/dist/css/bootstrap.min.css" rel="stylesheet"
+          integrity="sha384-+0n0xVW2eSR5OomGNYDnhzAbDsOXxcvSN1TPprVMTNDbiYZCxYbOOl7+AMvyTG2x" crossorigin="anonymous">
     <link href="{% static "js/prism_js/prism-tomorrow-night.css" %}" rel="stylesheet">
     <link rel="stylesheet" href="{% static "css/main.css" %}">
-    <script defer data-domain="grafeas.org" src="https://plausible.io/js/plausible.js"></script>
-
+    {% if not request.user.is_authenticated %}
+        {# only get metrics on folks who aren't staff #}
+        <script defer data-domain="grafeas.org" src="{% static "js/potentially.js" %}"></script>
```

Comment:
I'm a bit torn about this, If someone uses adblock, they probably don't want to be tracked, should we really use a workaround like this? How much do we really care about these stats being more accurate?

Reply:
Short version: yes.

Longer version: pretty much the only reason we're putting in metrics at all is because the numbers can directly influence funding; after we get the transcription app working, then it becomes "look how many hits our website gets! You should give us money to help us reach even more people!". According to plausible ([source](https://markosaric.com/google-analytics-blocking/)), somewhere between 6% and 26% of all loads are blocked, and that's enough to make me want to sidestep this. I'm an avid adblock user myself, and I also added some comments directly into the JS file in the PR. Since GitHub doesn't want to expand it, I wrote:

```
// plausible.js
// renamed file and host locally to bypass ublock origin
//
// If anyone reads this, please know that I care about privacy just as much as you.
// plausible.io is an ethical way to collect basic metrics.
// Everything we collect is 100% public; see it at https://plausible.io/grafeas.org
//
[minified script]
```

Reply:
Ok that's fair